lib/screens/app/app.dart

```diff
@@ -114,55 +127,137 @@ class _AppState extends State<App> with WidgetsBindingObserver {
       case AppLifecycleState.paused:
         break;
       case AppLifecycleState.resumed:
-        Provider.of<ConnectionNotifier>(context, listen: false).discoverEndpoints();
+        Provider.of<ConnectionNotifier>(context, listen: false)
+            .discoverEndpoints();
         break;
       case AppLifecycleState.detached:
         break;
     }
   }
 
+  void processIdentityRequest(request) async {
```

Comment:
this method is badass but why does request have no type?

Reply:
This method should not live inside the App.dart file

Create a service or utility class for it.

Reply:
I agree @7flash please change